packages/geostreaming/src/containers/Results/Pdf.jsx

```diff
@@ -0,0 +1,182 @@
+// @flow
```

Comment:
Not sure how this ended up here, but we can delete the whole `packages/geostreaming/src/containers/Results` folder.

Reply:
I removed the unused code. There were also some missing dependencies in `core__old`, which is now added.